recipes/7bitdi/all/conanfile.py

```diff
@@ -48,6 +51,7 @@ def configure(self):
             self.options.rm_safe("fPIC")
         if self.options.header_only:
             self.options.rm_safe("shared")
+            self.package_type = "header-library"
```

Comment:
AFAIK this is handled by conan automatically when the header_only option is set, but I can reference a test in the Conan main repo to make sure if you want! :)

Reply:
@RubenRBS 
Thank you for your review!
I removed this line.